src/components/GlobalScrollListener.tsx

```diff
@@ -3,7 +3,7 @@ import React, { useCallback } from 'react';
 import { useDispatch } from 'react-redux';
 
 import useScrollDirection, { ScrollDirection } from 'src/hooks/useScrollDirection';
-import { setIsBannerVisible } from 'src/redux/slices/banner';
+// import { setIsBannerVisible } from 'src/redux/slices/banner';
```

Comment:
I think we should remove all commented lines in this file since this change will be long-term